doc/manuals/user/walkthrough_apiv1.md

```diff
@@ -1011,31 +1001,51 @@ Let's examine in detail the different elements included in the payload:
     described [later in this document](duration.md#extending-duration).
     We are using "P1M" which means "one month".
 -   The notifyCondition element defines the "trigger" for
-    the subscription. There is a type element (which value in this case
-    is ONTIMERINTERVAL) and a condValueList element. The condValueList
-    element structure depends on the type. In the case of
-    ONTIMEINTERVAL, it includes exactly one condValue child element
-    whose value is a time interval (using again, as usual in NGSI, the
-    ISO 8601 format). A notification is sent with a frequency equal to
-    that interval. In the example above we are using 10 seconds
-    as interval.
+    the subscription. It uses the type ONCHANGE. The condValues vector contains a list of
+    attribute names. They define the
+    "triggering attributes", i.e. attributes that upon creation/change
+    due to [Entity Creation](#entity-creation) or
+    [Update context elements](#update-context-elements) trigger
+    the notification. The rule is that if at least one of the attributes
+    in the list changes (e.g. some kind of "OR" condition), then a
+    notification is sent. But note  that notification includes the
+    attributes in the attribute vector, which doesn't necessarily
+    include any attribute in the condValue. For example, in this case,
+    when Room1 pressure changes the Room1 temperature value is notified,
+    but not pressure itself. If you want also pressure to be notified,
+    the request would need to include
+    &lt;attribute&gt;pressure&lt;/attribute&gt; within the attribute vector
+    (or to use an empty attribute vector, which you already know means "all
+    the attributes in the entity"). Now, this example here, to be
+    notified of the value of *temperature* each time the value of
+    *pressure* changes may not be too useful. The example is chosen this
+    way only to show the enormous flexibility of subscriptions.
+-   The throttling element is used to specify a minimum
+    inter-notification arrival time. So, setting throttling to 5 seconds
+    as in the example above makes that a notification will not be sent
+    if a previous notification was sent less than 5 seconds ago, no
+    matter how many actual changes take place in that period. This is to
+    not stress the notification receptor in case of having context
```

Comment:
Or perhaps:

This is give the notification receptor a means to protect itself against context producers that update attribute values too frequently,